Dockerfile

```diff
@@ -7,6 +7,8 @@ RUN yum -y install \
 
 EXPOSE 80
 
+RUN mkdir -p /var/lib/php/session
```

Comment:
I'm speaking of the Dockerfile VOLUME syntax, which persists data across docker runs but not docker rm.